fakeserver/routes.go

```diff
@@ -167,14 +171,11 @@ func (s *server) routes() {
 		"/api/v4/apps/{a}/versions/{v}/builds/{b}/visitors/{id}/config",
 		getV4AppVisitorConfig,
 	)
+	// pretty sure this is deletable
```

Comment:
from what i can see, this endpoint was never used by mobile clients, rails client, js client, nor any other sort of client.